### PR TITLE
TELCODOCS-1231 - Add note to TALM documentation on handling of installPlans

### PR DESCRIPTION
**Pull request by slovern**
Version(s): 

- 4.13
- 4.12
- 4.11
- 4.10

Issue: https://issues.redhat.com/browse/TELCODOCS-1231

Link to docs preview: https://57857--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-apply-policies_cnf-topology-aware-lifecycle-manager

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc [hunk 1]
```diff
@@ -358,6 +358,12 @@ NAMESPACE                              NAME            CSV
 openshift-logging                      install-6khtw   cluster-logging.5.3.3-4             Manual     true <1>
 ----
 <1> The install plans have their `Approval` field set to `Manual` and their `Approved` field changes from `true` to `false` after {cgu-operator} approves the install plan.
```
Comment (jc-rh) [inline modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc:360]: should be false to true

Comment (slovern) [reply modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc]: thanks. Updated and added a note to the JIRA.

File: modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc [hunk 2]
```diff
@@ -357,7 +357,13 @@ $ oc get installplan -n <subscription_namespace>
 NAMESPACE                              NAME            CSV                                 APPROVAL   APPROVED
 openshift-logging                      install-6khtw   cluster-logging.5.3.3-4             Manual     true <1>
 ----
-<1> The install plans have their `Approval` field set to `Manual` and their `Approved` field changes from `true` to `false` after {cgu-operator} approves the install plan.
+<1> The install plans have their `Approval` field set to `Manual` and their `Approved` field changes from `false` to `true` after {cgu-operator} approves the install plan.
++
+[NOTE]
+====
+When {cgu-operator} is remediating a policy containing a subscription, it automatically approves any install plans attached to that subscription. 
+Where multiple install plans are needed to get the operator to the latest known version, {cgu-operator} may approve multiple install plans, upgrading through one or more intermediate versions to get to the final version.
```
Comment (tmalove) [inline modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc:365]:
```suggestion
Where multiple install plans are needed to get the operator to the latest known version, {cgu-operator} might approve multiple install plans, upgrading through one or more intermediate versions to get to the final version.
```
See SSG: https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=Avoid+these+words

**Review by tmalove — APPROVED** [review]

**Comment by slovern** [comment]
/ label telco

**Pull request merged**